### ISSUE TITLE
AttributeError: 'Molecule' object has no attribute 'get_scene'

### ISSUE THREAD

**Issue by hududed** [materialsproject/crystaltoolkit#108]
Hi

following the [workshop](https://github.com/materialsproject/workshop/blob/master/lessons/pymatgen/1%20-%20pymatgen%20core%20use.ipynb). Got the error 
`AttributeError: 'Molecule' object has no attribute 'get_scene'` when I try to `view(molecule)`.

Edit: This error shows up for the CO and HCN molecule examples, but not for structure.

Cheers, Hud

**Comment by mkhorton** [comment]
Hi @hududed, Crystal Toolkit is still pre-release (see docs.crystaltoolkit.org).

If you just want to get something working, install the latest crystal toolkit + use `from crystal_toolkit.renderables.sitecollection import *` and it will likely work.

A note on the workshop; we included Crystal Toolkit simply to make it easier to visualize what we were doing as we went along. It's not necessary for running any of the tutorials and you're welcome to export your molecule as `.xyz` or similar to view in a program of your choice.

I'm working hard to get an official release out for this, so stay tuned :-)
